client/src/pages/ManageBookingPage/ManageBookingPage.js

```diff
@@ -21,20 +33,49 @@ const useStyles = makeStyles((theme) => ({
 
 function ManageBookingPage(props) {
   const classes = useStyles();
-  const [date, changeDate] = useState(new Date());
-  const [requests, setRequests] = useState([]);
+  const [bookings, setBookings] = useState([]);
 
   useEffect(() => {
     axios.get(`/request/${props.userID}`).then(({ data }) => {
-      setRequests(data);
+      setBookings(data);
```

Comment:
are these already sorted by date when they're returned?

Reply:
Yes, sorted date is returned from mongo